SPECS/kernel/config_aarch64

```diff
@@ -3401,6 +3393,7 @@ CONFIG_NET_VENDOR_MICROSEMI=y
 CONFIG_MSCC_OCELOT_SWITCH_LIB=m
 CONFIG_MSCC_OCELOT_SWITCH=m
 CONFIG_NET_VENDOR_MICROSOFT=y
+CONFIG_MICROSOFT_MANA=m
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/e5f84fbfb236129c180c1499013a2dd1cbd634e9